UI/song

ssonghj opened the pull request.

ssonghj committed:
feat : myPage ui 일괄 추가

ssonghj committed:
프로필 등록 xml 추가

SseO-KR reviewed (commented):
1. customToolbar 변경 부탁드립니다
2. CircleImageView id의 경우 civ로 통일 부탁드립니다.
3. tv_profile_email_value의 경우 tv_user_email 이 더 직관적이지 않을까요??
4. 두번째 ConstraintLayout에 tools:context가 딸려온 것 같습니다
5. marginStart,marginTop 등 @dimen 사용해야할 것 같습니다.
6. btn_modify 변수명 수정해야할 것 같습니다

SseO-KR reviewed (commented):
다음에는 커밋, 리퀘스트 쪼개서 부탁드립니다 ㅠㅜ

EditProfileFragment, MyPageFragment, OthersFragment
1. FragmentHomeBinding으로 되어있는데 다른 걸로 바꿔야하지 않을까요??
2. onCreate 함수는 필요 없어보입니다

fragment_achievement, fragment_edit_profile
1. margin에서 @dimens 사용해야할 것 같습니다
2. imageView 경우 iv로 통일해주셔야할 것 같습니다

fragment_my_board
1. margin에서 @dimens 사용해야할 것 같습니다

fragment_my_page
1. customtoolbar 사용이 필요해보입니다
2. margin에서 @dimens 사용해야할 것 같습니다
3. imageView 경우 iv로 통일해주셔야할 것 같습니다
4. CircleImageView 경우 civ로 통일해주셔야할 것 같습니다.
5. dimens 사용이 필요해보입니다.
6. recyclerview 변수명

fragment_others
1. customtoolbar 사용이 필요해보입니다
2. @dimens 사용해야할 것 같습니다

fragment_user_detail
1. customtoolbar 사용이 필요해보입니다
2. @dimens 사용해야할 것 같습니다
3. ImageView는 iv, CircleImageview는 civ로 통일해야할 것 같습니다
4. tv_user_point 경우 ImageView로 되어있습니다.

item_my_board
1. @dimens 사용해야할 것 같습니다
2. ImageView iv로 통일해야할 것 같습니다

item_my_crew_history
1. @dimens 사용해야할 것 같습니다
2. ImageView iv로 통일해야할 것 같습니다

item_my_running_record
1. @dimens 사용해야할 것 같습니다
2. ImageView iv로 통일해야할 것 같습니다

my_calendar_header
1. 변수명 legend 대신 dayOfWeek 또는 tv_sunday 이런식으로 사용하는 것 어떤가요

ssonghj committed:
코멘트 반영

EditProfileFragment, MyPageFragment, OthersFragment
- Binding명 변경
- onCreate 함수 삭제

fragment_achievement, fragment_edit_profile
- iv로 통일

fragment_my_page
- customtoolbar 사용
- imageView -> iv 변경
- CircleImageView -> civ로 변경

fragment_others
- customtoolbar 사용

fragment_user_detail
- customtoolbar 사용
- imageView -> iv 변경
- CircleImageView -> civ로 변경
- tv_user_point -> iv_user_point 변경

item_my_board
- imageView -> iv 변경

item_my_crew_history
- imageView -> iv 변경

item_my_running_record
- imageView -> iv 변경

my_calendar_header
- legend -> dayOfWeek로 변경

SseO-KR committed:
Update build.gradle

SseO-KR committed:
Update HomeFragment.kt

SseO-KR committed:
Update HomeFragment.kt

SseO-KR committed:
Update colors.xml

SseO-KR committed:
Update dimen.xml

SseO-KR committed:
Update colors.xml

SseO-KR committed:
Update themes.xml

SseO-KR committed:
fix : conflict 해결

SseO-KR committed:
fix : conflict 해결
